src/framework/theme/components/calendar-kit/components/calendar-week-number/calendar-week-number.component.ts

```diff
@@ -11,10 +11,12 @@ import { NbCalendarSize } from '../../model';
 @Component({
   selector: 'nb-calendar-week-numbers',
   template: `
-    <div class="sign">{{ weekNumberSymbol }}</div>
-    <div class="week-cell" *ngFor="let week of getWeeks()">{{ week }}</div>
+    <div class="sign-container">
```

Comment:
Do we really need an additional container here? Are you sure we can't style it properly without it?

Reply:
> Do we really need an additional container here?

An additional container is required here because the sign cell should be the same height as weekdays row on the right. Weekday row height is a divider (border) width + height of the cell inside. I found, having the sign container with a border + .sign inside simpler and less error-prone than using calc for height.